### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Just ``Linux`` is passing for now, I will disable windows and travis for Linux
I will let travis just to generate docs and test flake8

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      conda config --set always_yes yes
+      conda config --set auto_update_conda False
+      conda info
+      # avoids a python 3.7 problem
+      conda install -q cytoolz
+      if [[ "$PYTHON_VERSION" == "2.7" ]]; then
+        conda install -q futures scandir;
+      fi
+      conda install -q anaconda-client git requests filelock contextlib2 jinja2 patchelf ripgrep pyflakes beautifulsoup4 chardet pycrypto glob2 psutil pytz tqdm conda-package-handling py-lief
+      conda install -c conda-forge pytest-azurepipelines -y
```
Comment (marcelotrevisani) [inline azure-pipelines.yml]: It would be good to have ``pytest-azurepipelines`` on the main channels

**Comment by marcelotrevisani** [comment]
Related to 

- https://github.com/conda/conda-build/pull/3150
- https://github.com/conda/conda-build/issues/3658

**Comment by msarahan** [comment]
Thanks.  We have really needed this.

**Comment by msarahan** [comment]
pytest-azurepipelines in defaults is on the way.

**Pull request merged**